modules/local/pyopenms_idfilter.nf

```diff
@@ -0,0 +1,34 @@
+process PYOPENMS_IDFILTER {
```

Comment:
Why is this PYOPENMS and not with Openms?

Reply:
See above